sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/Azure.AI.Vision.ImageAnalysis.Tests.csproj

```diff
@@ -5,6 +5,11 @@
     <!-- We don't care about XML doc comments on test types and members -->
     <NoWarn>$(NoWarn);CS1591</NoWarn>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="SessionRecords\**" />
+    <EmbeddedResource Remove="SessionRecords\**" />
+    <None Remove="SessionRecords\**" />
+  </ItemGroup>
```

Comment:
I think VS adds this - but these directories shouldn't exist anymore. I think these can be manually removed.